### PR TITLE
docs: more feature details, compare against WC

### PR DESCRIPTION
**Pull request by dholbach**
- fix Helm link
 - extend features to point to feature discussion between
   flux and WC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,13 +40,32 @@ Flux is most useful when used as a deployment tool at the end of a
...
 
 - [Automated git → cluster synchronisation](/site/introduction.md#automated-git-cluster-synchronisation)
 - [Automated deployment of new container images](/site/introduction.md#automated-deployment-of-new-container-images)
-- [Integrations with other devops tools](/site/introduction.md#integrations-with-other-devops-tools) ([Helm](site/helm/helm-integration.md) and more)
+- [Integrations with other devops tools](/site/introduction.md#integrations-with-other-devops-tools) ([Helm](/site/helm/helm-integration.md) and more)
 - No additional service or infrastructure needed - Flux lives inside your
   cluster
+- Straight-forward control over the state of deployments in the
+  cluster (rollbacks, lock a specific version of a workload, manual
+  deplomyents, )
```
Comment (squaremo) [inline README.md]: stray comma?

File: README.md [hunk 2]
```diff
@@ -40,13 +40,32 @@ Flux is most useful when used as a deployment tool at the end of a
...
+- Observability: Audit trail (who did what when) and leave messages as
```
Comment (squaremo) [inline README.md]: "git commits are an audit trail, and you can record e.g., why a given deployment was locked.

File: README.md [hunk 3]
```diff
@@ -40,13 +40,32 @@ Flux is most useful when used as a deployment tool at the end of a
...
+  to why a given deployment was locked
+
+Weave Cloud builds on top of the great work in Flux and is a SaaS
```
Comment (squaremo) [inline README.md]: "Weave Cloud is a SaaS product by Weaveworks that includes Flux, as well as"

File: README.md [hunk 4]
```diff
@@ -40,13 +40,32 @@ Flux is most useful when used as a deployment tool at the end of a
...
+product by Weaveworks that gives you
+
+ - a UI and alerts for deployments: nicely integrated overview, all flux
+   operations just a click away.
+ - full observability and insights into your cluster: Instantly start using
+   monitoring dashboards for your cluster, hosted 13 months of history, use
+   a realtime map of your cluster to debug and analyse its state.
+
+There is lots more, so if you want to learn more about Weave Cloud and see
```
Comment (squaremo) [inline README.md]: "If you want to learn more about Weave Cloud, you can see it in action on its homepage." (is this accurate, can you see it in action on the website?)

Comment (dholbach) [reply README.md]: bildschirmfoto von 2018-06-01 16-05-29
Looks like it. :-)

**Comment by dholbach** [comment]
Thanks a lot for the feedback, Michael!

File: README.md [hunk 5]
```diff
@@ -40,13 +40,34 @@ Flux is most useful when used as a deployment tool at the end of a
...
 
 - [Automated git → cluster synchronisation](/site/introduction.md#automated-git-cluster-synchronisation)
 - [Automated deployment of new container images](/site/introduction.md#automated-deployment-of-new-container-images)
-- [Integrations with other devops tools](/site/introduction.md#integrations-with-other-devops-tools) ([Helm](site/helm/helm-integration.md) and more)
+- [Integrations with other devops tools](/site/introduction.md#integrations-with-other-devops-tools) ([Helm](/site/helm/helm-integration.md) and more)
 - No additional service or infrastructure needed - Flux lives inside your
   cluster
+- Straight-forward control over the state of deployments in the
+  cluster (rollbacks, lock a specific version of a workload, manual
+  deplomyents)
```
Comment (squaremo) [inline README.md]: tyop!

**Pull request merged**